accepted/2023/feature-switch-attributes.md

```diff
@@ -0,0 +1,766 @@
+# Attribute-based model for feature switches
```

Comment:
The introduction (all the text until the Background header) is very hard to read because it mixes defining terms and listing existing functionality with problem statement.

I think you'll want to describe (in simple non-jargon terms) what UX currently exists, what problem it has, and what (at a high level) you're going to propose to address this.